temporian/implementation/numpy_cc/operators/calendar.cc

```diff
@@ -0,0 +1,258 @@
+#include <pybind11/numpy.h>
+#include <pybind11/pybind11.h>
+
+#include "absl/time/time.h"
+
+#include "temporian/implementation/numpy_cc/operators/common.h"
+
+namespace {
+namespace py = pybind11;
+
+static absl::TimeZone parse_tz(const py::object tz) {
```

Comment:
Why "static" ? (here and in other places)

Reply:
That was a workaround when I had `calendar.h` I will remove it now